src/test/java/uk/gov/justice/digital/hmpps/vsip/cucumber/steps/SelectVisitorsStep.java

```diff
@@ -39,4 +39,20 @@ public void iSelectAVisitorFormTheListFromBristolEstablishment() {
     public void iSelectAVisitorFormTheListForAPrisoner() {
         selectVisitorsPage.choosePrisonerB();
     }
+
+    @And("I select a visitr form the list from Bristol establishment")
```

Comment:
spelling

Reply:
Covered as part of Vb 3308 additional tests